File: src/VetrinaDolci.WebAPI/Models/IngredientiDolce.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VetrinaDolci.WebAPI.Models
{
    public class IngredientiDolce
    {
        public int DolceId { get; set; }
        public Dolce Dolce { get; set; }
        public int IngredienteId { get; set; }
        public Ingrediente Ingrediente { get; set; }

        public int Quantita { get; set; }
        public string UnitaDiMisura { get; set; }
    }
}

File: src/vetrina-dolci-client/src/app/models/ingredienti-dolce.ts
import { Dolce } from "./dolce";
import { Ingrediente } from "./ingrediente";

export class IngredientiDolce {
    id: number;
    quantita: string;
    unitaDiMisura: string;
    note: string;

    dolceId: number;
    dolce: Dolce;

    ingredienteId: number;
    ingrediente: Ingrediente;

    constructor(data?: Partial<IngredientiDolce>) {
      Object.assign(this, data);
    }
  }
  